### PR TITLE
Clarify that the latest Python might not work

### PR DESCRIPTION
**Pull request by Slapbox**
I wasted a lot of time not realizing Python 3.11 isn't supported because of the lack of clarity in the line I've edited.

##### Checklist
- [x] documentation is changed or added
- [x] commit message follows [commit guidelines](https://github.com/googleapis/release-please#how-should-i-write-my-commits)

##### Description of change
Remove unclear reference to "current" Python version, that does not necessarily mean "current."

Someone please let me know if I need to re-title my PR. I don't see the relevant prefix, but maybe "docs:" ?

### CODE REVIEW

**Comment by Slapbox** [comment]
Actually, I get the same error no matter what version of Python 3 I use. Why is that? Isn't this issue supposed to be fixed already?

https://github.com/nodejs/node-gyp/issues/2219

**Comment by cclauss** [comment]
As stated above, the problem was not related to the Python version.   This change is not needed because we work with and test pre-release versions of Python to ensure that our installation process is compatible with them on the day that they are released.

**Comment by Slapbox** [comment]
@cclauss I looked through the 8 `binding.gyp` files in our `node_modules` and none of them appear to specify `rU` or `Ur`, and `node-gyp` is at `9.3.0` which is the latest as of today.

I used Python2.7 to get around this, but I don't understand why the issue is occurring.

> Please look thru your error log for the string `gyp info using node-gyp@`

Can you clarify what you mean by "your error log"? What would this be on a Windows machine? I don't see an `npm-debug.log` and there doesn't appear to be anything in Event Viewer (not that I expected there would be.)

**Comment by cclauss** [comment]
1. What command did you type?
2. What text appeared on your screen after you typed it?

**Comment by Slapbox** [comment]
Well, I just realize that `node-gyp` is installed locally in our `node_modules`, and that one is _not_ the latest version - so I guess that's the issue.

The failing module was `fibers`, which our upcoming version doesn't rely on at all - so I guess this is moot. It also explicitly says it's incompatible with Node 16, which I didn't know.

Thanks very much for your help @cclauss!

In case you still had some interest in the output upon failure, here it is. The command we ran was just `yarn`.

```
error C:\Projects\MyProject\node_modules\fibers: Command failed.
Exit code: 1
Command: node build.js || nodejs build.js
Arguments:
Directory: C:\Projects\MyProject\node_modules\fibers
Output:
gyp info it worked if it ends with ok
gyp info using node-gyp@7.1.2
gyp info using node@16.18.0 | win32 | x64
gyp info find Python using Python version 3.11.0 found at "C:\Python311\python.exe"
gyp info find VS using VS2019 (16.11.33027.164) found at:
gyp info find VS "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools"
gyp info find VS run with --verbose for detailed information
(node:5036) [DEP0150] DeprecationWarning: Setting process.config is deprecated. In the future the property will be read-only.
(Use `node --trace-deprecation ...` to show where the warning was created)
gyp info spawn C:\Python311\python.exe
gyp info spawn args [
gyp info spawn args   'C:\\Projects\\MyProject\\node_modules\\node-gyp\\gyp\\gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'msvs',
gyp info spawn args   '-I',
gyp info spawn args   'C:\\Projects\\MyProject\\node_modules\\fibers\\build\\config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   'C:\\Projects\\MyProject\\node_modules\\node-gyp\\addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   'C:\\Users\\Me\\AppData\\Local\\node-gyp\\Cache\\16.18.0\\include\\node\\common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=C:\\Users\\Me\\AppData\\Local\\node-gyp\\Cache\\16.18.0',
gyp info spawn args   '-Dnode_gyp_dir=C:\\Projects\\MyProject\\node_modules\\node-gyp',
gyp info spawn args   '-Dnode_lib_file=C:\\\\Users\\\\Me\\\\AppData\\\\Local\\\\node-gyp\\\\Cache\\\\16.18.0\\\\<(target_arch)\\\\node.lib',
gyp info spawn args   '-Dmodule_root_dir=C:\\Projects\\MyProject\\node_modules\\fibers',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'C:\\Projects\\MyProject\\node_modules\\fibers\\build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
Traceback (most recent call last):
  File "C:\Projects\MyProject\node_modules\node-gyp\gyp\gyp_main.py", line 51, in <module>
    sys.exit(gyp.script_main())
             ^^^^^^^^^^^^^^^^^
  File "C:\Projects\MyProject\node_modules\node-gyp\gyp\pylib\gyp\__init__.py", line 670, in script_main
    return main(sys.argv[1:])
           ^^^^^^^^^^^^^^^^^^
  File "C:\Projects\MyProject\node_modules\node-gyp\gyp\pylib\gyp\__init__.py", line 662, in main
    return gyp_main(args)
           ^^^^^^^^^^^^^^
  File "C:\Projects\MyProject\node_modules\node-gyp\gyp\pylib\gyp\__init__.py", line 629, in gyp_main
    [generator, flat_list, targets, data] = Load(
                                            ^^^^^
  File "C:\Projects\MyProject\node_modules\node-gyp\gyp\pylib\gyp\__init__.py", line 150, in Load
    result = gyp.input.Load(
             ^^^^^^^^^^^^^^^
  File "C:\Projects\MyProject\node_modules\node-gyp\gyp\pylib\gyp\input.py", line 3021, in Load
    LoadTargetBuildFile(
  File "C:\Projects\MyProject\node_modules\node-gyp\gyp\pylib\gyp\input.py", line 411, in LoadTargetBuildFile
    build_file_data = LoadOneBuildFile(
                      ^^^^^^^^^^^^^^^^^
  File "C:\Projects\MyProject\node_modules\node-gyp\gyp\pylib\gyp\input.py", line 239, in LoadOneBuildFile
    build_file_contents = open(build_file_path, "rU").read()
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: invalid mode: 'rU' while trying to load binding.gyp
gyp ERR! configure error
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (C:\Projects\MyProject\node_modules\node-gyp\lib\configure.js:351:16)
gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:293:12)
gyp ERR! System Windows_NT 10.0.22000
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Projects\\MyProject\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild" "--release"
gyp ERR! cwd C:\Projects\MyProject\node_modules\fibers
gyp ERR! node -v v16.18.0
gyp ERR! node-gyp -v v7.1.2
gyp ERR! not ok
node-gyp exited with code: 1
Please make sure you are using a supported platform and node version. If you
would like to compile fibers on this machine please make sure you have setup your
build environment--
Windows + OS X instructions here: https://github.com/nodejs/node-gyp
Ubuntu users please run: `sudo apt-get install g++ build-essential`
RHEL users please run: `yum install gcc-c++` and `yum groupinstall 'Development Tools'`
```

**Comment by cclauss** [comment]
Line 8:
> gyp info using node-gyp@7.1.2

https://github.com/nodejs/node-gyp/releases

**Comment by Slapbox** [comment]
Yeah I realized, I mentioned - but I'm sure you're very busy and I know I'm a bit wordy. I appreciate all your help. Thanks again!
